services/likes_service.py

```
@@ -0,0 +1,63 @@
+            "like_id": str(uuid.uuid4()),
+            "user_id": user_id,
+            "target_type": like_in.target_type,
+            "target_id": like_in.target_id,
+            "created_at": datetime.now(timezone.utc).isoformat()
+        }
+        likes.append(new_like_data)
+
+    print(likes)
+    save_likes(likes)
+
+    current_total = len([l for l in likes if l["target_id"] == like_in.target_id])
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)